src/tests/strfmt.rs

```diff
@@ -130,6 +134,10 @@ fn test_values() {
         ("{x:<4d}", "", 3),
         ("{x:,}", "", 3),
         ("{x:<-10}", "", 3),
+
+        // TODO
```

Comment:
remove TODO?

Reply:
These are following the pattern of tests for not-yet-supported functionality that you find in the float and uint tests. They produce `sign aware zero padding and Align \'=\' not yet supported`.

Reply:
I think that TODO is already in the code though? I would remove this one (these tests will fail when that is implemented, but that is okay).

Reply:
I feel like some comment to categorize them makes sense. Does `not yet supported` work?

Reply:
Yes, that would be fine. Feel free to change if it bothers you (personally I think it is fine, it was a minor nit).